Project/Source/Modules/ModuleUserInterface.h

```diff
@@ -31,6 +31,6 @@ class ModuleUserInterface : public Module {
 	void CreateQuadVBO(); // Creates a vbo made by two triangles centered that form a Quad
 
 private:
-	ComponentEventSystem* currentEvSys = nullptr;
+	UID currentEvSys = 0;
```

Comment:
No need to change, but as a note.
With Jaime's refactor we can now safely store Component pointers.
The only exception would be that that pointer has to be serialized, then yes, use UIDs (not sure if that is the case here).

Reply:
Didn't know, roger that!